src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -517,13 +522,48 @@ const OfflineTokenRow = ({ offlineToken, onValueChanged, formValidationFailed })
                 </FlexItem>
             </Flex>,
         },
+        EXPIRED: {
+            option: "default",
+            message: <Flex>
+                <FlexItem grow={{ default: 'grow' }}>{_("Old token expired") + " "}</FlexItem>
+                <FlexItem>
+                    { link }
+                    {" " + _("Then copy and paste it above.")}
+                </FlexItem>
+            </Flex>,
+        },
         SUCCESS: {
             option: "success",
             message: _("Valid token"),
         },
     };
 
     const [validationState, setValidationState] = useState(validationStates.DEFAULT);
+    const [disabled, setDisabled] = useState(false);
+
+    useEffect(() => {
+        loadOfflineToken((token) => {
+            if (token) {
+                onValueChanged("offlineToken", token);
+                setDisabled(true);
+                setValidationState(validationStates.INPROGRESS);
+                getAccessToken(token)
+                        .then(out => {
+                            const accessToken = out.trim();
+                            setDisabled(false);
+                            onValueChanged("accessToken", accessToken);
+                            setValidationState(validationStates.SUCCESS);
+                        })
+                        .catch(ex => {
+                            setValidationState(validationStates.EXPIRED);
+                            onValueChanged("offlineToken", "");
```

Comment:
It comes from a previous design discussion: https://github.com/cockpit-project/cockpit-machines/pull/774#issuecomment-1210577729

If we just go directly to a DEFAULT state, then the process would look like this: User opens a dialog, there would be a greyed-out area with a old token visible for a few seconds while verification is happening (it turns out it's expired), then UI would empty the offline token field. I believe in that case, it might be confusing, and user might thing "what just happend to an old token I had saved there?". It's not obvious knowledge that tokens expire after 30 days of inactivity.